en/simulation/README.md

```diff
@@ -48,23 +48,23 @@ Message | Direction | Description
 [HIL_RC_INPUTS_RAW](http://mavlink.org/messages/common#HIL_RC_INPUTS_RAW) | Sim to PX4 | The RAW values of the RC channels received.
 
 
-## Standard MAVLink UDP Ports
+## Default PX4 MAVLink UDP ports
 
-PX4 defines a number of standard UDP broadcast ports that ground control stations (e.g. *QGroundControl*, Offboard APIs and simulators) can listen to in order to connect to the autopilot:
+By default, PX4 uses well-defined MAVLink UDP ports for communication with ground control stations (e.g. *QGroundControl*), Offboard APIs (e.g. DroneCore, MAVROS) and simulator APIs (e.g. Gazebo). These ports are:
 
-- Offboard APIs: 14540
-- Ground Control Stations: 14550  (*QGroundControl* automatically connects to PX4 broadcasting on this port).
-- Simulators: 14560
+* Port **14540** is used for communication with offboard APIs. PX4 broadcasts data towards this port, and Offboard APIs are expected to listen.
```

Comment:
@cyberang3l  Might the first two be better as below (removing "broadcast" removes the "feeling" that this is single direction communication).

* Port **14540** is used for communication with offboard APIs. Offboard APIs are expected to listen for connections on this port. 
* Port **14550** is used for communication with ground control stations. GCS are expected to listen for connections on this port. *QGroundControl* listens to this port by default.

Reply:
I agree with this. Better remove `broadcast`.